src/main/resources/assets/offershud/lang/zh_cn.json

```diff
@@ -0,0 +1,9 @@
+{
+  "text.autoconfig.offershud.title": "Offers HUD",
+  "text.autoconfig.offershud.option.enabled": "启用",
+  "text.autoconfig.offershud.option.ignoreNoProfession": "忽略无业村民",
+  "text.autoconfig.offershud.option.suppressVillagerHeadRolling": "Suppress villager head rolling",
```

Comment:
What about this line?
This feature prevents unemployed villagers from tilting their heads when interacted with.

...I haven't checked it for a long time, so maybe you didn't see this behavior? If so, maybe the function itself is unnecessary.

Reply:
Ummm.., I don't know.